develop/tutorials/articles/110-portlets/15-javascript/04-npm/04-structure-of-osgi-bundles-npm.markdown

```diff
@@ -6,43 +6,43 @@ and modify them to work with @product@'s [Liferay AMD Loader](https://github.com
 The liferay-npm-bundler automates this process for you, creating a bundle 
 similar to the one below:
 
-    - `my-bundle/`
-        - `META-INF/`
-            - `resources/`
-                - `package.json`
-                    name: my-bundle-package
-                    version: 1.0.0
-                    main: lib/index
-                    dependencies:
-                        isarray: 2.0.0
-                        isobject: 2.1.0
-                    ...
-                - `lib/`
-                    -`index.js`
-                    ...
+- `my-bundle/`
+    - `META-INF/`
+        - `resources/`
+            - `package.json`
+                name: my-bundle-package
```

Comment:
@sez11a I have a small formatting issue with the contents of the package.json file in the folder structure. Do you know how to handle this? It's formatted the same way in each tutorial that references it.